crates/core/src/repository/mod.rs

```diff
@@ -228,7 +271,7 @@ impl ReleaseChannel {
 
 impl Default for ReleaseChannel {
     fn default() -> ReleaseChannel {
-        ReleaseChannel::Stable
+        ReleaseChannel::Default(GlobalReleaseChannel::Stable)
```

Comment:
@tarkah I am running in circles here.
If we don't embed `GlobalReleaseChannel` here we have to pass a reference to `Config` every time, and it's used A LOT of places.
If we pass it like i have now we would need to set it on every parse. I would like some help/feedback here. What is the right play?